clar2wasm/src/words/sequences.rs

```diff
@@ -46,6 +47,8 @@ impl ComplexWord for ListCons {
                 )));
             };
 
+        self.charge(generator, builder, num_elem)?;
```

Comment:
Isn't it wrong for the cases where the number of elements in the list is smaller than the max number of elements?